local/build-binary

```diff
@@ -61,6 +62,29 @@ fi
 
 wget_loop "googletest-release-${GMOCK_VERSION}.zip" "https://github.com/google/googletest/archive/release-${GMOCK_VERSION}.zip"
 
+if [[ ${WITH_BORINGSSL} == "true" ]]; then
+    # ------------------------------------------------------------------------------
+    # compile boringssl libs
+    # ------------------------------------------------------------------------------
+    git clone https://github.com/google/boringssl.git
+    mkdir boringssl/bin
+    cd boringssl/bin
+    if [[ -f /etc/redhat-release ]]; then
+        source /opt/rh/go-toolset-7/enable
```

Comment:
go-toolset is installed for rhel7. So on centos6 and centos8 there will be an issue

Reply:
Added support for RHEL8 and added missing package for unwind tests

And added a check for centos 6, as BoringSSL can't compile on it